### PR TITLE
Added new course btn, sort by and search section

### PR DESCRIPTION
**Pull request by MariaAfonina**
Added new course btn and handle click to change route.
Also, sort by dropdown and search input.

Changed name of GetOffersP**r**arams => GetOffersParams

Screenshot 2023-10-20 at 15 57 24

### CODE REVIEW

File: src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { useTranslation } from 'react-i18next'
+import AddIcon from '@mui/icons-material/Add'
+import SearchIcon from '@mui/icons-material/Search'
+import Box from '@mui/material/Box'
+
+import AppButton from '~/components/app-button/AppButton'
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+
+import { authRoutes } from '~/router/constants/authRoutes'
+import CoursesFilterBar from '~/containers/find-course/courses-filter-bar/CoursesFilterBar'
```
Comment (Tolik170) [inline src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx:12]: locate containers with components

File: src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { styles } from '~/containers/my-courses/add-course-with-input/AddCourseWithInput.styles'
+
+interface AddCourseWithInputProps {
+  button?: ReactElement
```
Comment (Tolik170) [inline src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx:16]: Remove this interface, for now, it is redundant

File: src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+const AddCourseWithInput: FC<AddCourseWithInputProps> = () => {
+  const { t } = useTranslation()
+
+  const onClear = () => {
+    console.log('Hello')
```
Comment (Tolik170) [inline src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx:23]: you can already create state for input and clear it

File: src/pages/my-courses/MyCourses.styles.ts [hunk 4]
```diff
@@ -4,5 +4,11 @@ export const styles = {
   title: {
     typography: TypographyVariantEnum.H4,
     mb: '40px'
+  },
+  tabs: {
```
Comment (Tolik170) [inline src/pages/my-courses/MyCourses.styles.ts:8]: not correct naming at all, maybe better name it as divider?

File: src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx (hunk 1 above)
Comment (MariaAfonina) [reply src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx:12]: Done

File: src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx (hunk 2 above)
Comment (MariaAfonina) [reply src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx:16]: I saw this but forgot to change it, thanks

**Pull request closed without merging**